iidm/iidm-tck/src/test/java/com/powsybl/iidm/network/tck/AbstractBatteryTest.java

```diff
@@ -130,7 +130,7 @@ public void testRemove() {
         int count = network.getBatteryCount();
         Battery battery = network.getBattery(TO_REMOVE);
         assertNotNull(battery);
-        battery.remove();
+        battery.remove();    // <---- TODO HERE
         assertNotNull(battery);
         Terminal terminal = battery.getTerminal();
```

Comment:
On network-store-store's side : battery.getTerminal() works but then battery.getTerminal().isConnected() raise exception with message "Object has been removed in current variant" because the resource of the terminal instance is null